libraries/classes/Server/Privileges.php

```diff
@@ -5131,7 +5131,7 @@ public static function addUserAndCreateDatabase(
     public static function getHashedPassword($password)
     {
         $result = $GLOBALS['dbi']->fetchSingleRow(
-            "SELECT PASSWORD('" . $password . "') AS `password`;"
+            "SELECT PASSWORD('" . addslashes($password) . "') AS `password`;"
```

Comment:
You could use `addcslashes($password, '"');` ? (I did not try it but seems more appropriate since only `"` must be escaped

https://www.php.net/manual/en/function.addcslashes.php

```suggestion
            "SELECT PASSWORD('" . addcslashes($password, '"') . "') AS `password`;"
```

Reply:
@williamdes, I checked by making the change, but it is not working as the issue is not being resolved and it shows Password as No in User Accounts

Reply:
@kartik1000 okay, I removed my suggestion
```sql
SELECT PASSWORD(''),PASSWORD('"'),PASSWORD('\''),PASSWORD("'"),PASSWORD("\"")
```

Reply:
@kartik1000 here is a better one: https://github.com/phpmyadmin/phpmyadmin/pull/15428/files#r312753043

Reply:
![image](https://user-images.githubusercontent.com/43112279/62847948-ac49b880-bcf6-11e9-83d2-934464c88478.png)
The error I am getting after making the change.

Reply:
Use `$GLOBALS['dbi']->` instead :)

Reply:
This one is working @williamdes :) Shall I make the change then?

Reply:
Yes, absolutely :)